### DISCUSSION TITLE
Relative path function

### DISCUSSION THREAD

**Discussion by Natejoestev** [tauri-apps/tauri#8270] [Ideas]
I am making a program that manipulates files and shows the path to the user but I don't need the absolute path.

It would be nice if the `path` api had a `relative` function:
```js
import { relative } from '@tauri/api/path';

relative('C:/users', 'C:/users/a/b/c.txt') == './a/b/c.txt';
relative( 'C:/taco', 'C:/users/a/b/c.txt') == '../users/a/b/c.txt';
```

This would be useful for representing paths to the user. And sometimes you need to get a relative path for other stuff.
PS. I know you could just hardcode it into your project. But then you have to do that in every project you need it.

**Comment by frankolson** [comment]
I also think this would be a useful feature. Similar to the Node.js path module's `relative` function: https://nodejs.org/api/path.html#pathrelativefrom-to
